### DISCUSSION TITLE
Notes on breaking changes in Verify 17

### DISCUSSION THREAD

**Discussion by bmueller84** [VerifyTests/Verify#535] [General]
Hi and thanks for this great project! I've been using it for about 2 years now and am pretty satisfied. Thanks for the great work!

Now, one thing that bothered me today where the breaking changes in Verify 17. I updated the nuget and was hit by the fact that ModifySerializer went away (which turned out to be an easy fix) and that the behavior w.r.t. ignoring bools changed. I was able to adapt by now, but it took me much longer than necessary because I couldn't find any release notes on the breaking changes, and had to look into the git history to understand what has changed. Did I miss something? If yes, I'd really appreciate if you could make such notes more discoverable. If not, what's your plan on providing such information?

Thanks again!

**Comment by SimonCropp** [comment]
release notes, and hot to handle braking changes are in the milestone https://github.com/VerifyTests/Verify/issues?q=is%3Aclosed+milestone%3A17.0.0

for example the ModifySerializer change https://github.com/VerifyTests/Verify/pull/533

**Comment by SimonCropp** [comment]
the release notes are also duplicated in the GH Releases https://github.com/VerifyTests/Verify/releases/tag/17.0.0

**Reply by bmueller84** [reply]
Great, thanks!
